2018-09-11 02:23:24 Error: [PDOException] SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Database\Driver.php on line 92
Stack Trace:
#0 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Database\Driver.php(92): PDO->__construct('mysql:host=loca...', 'my_app', 'secret', Array)
#1 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Database\Driver\Mysql.php(106): Cake\Database\Driver->_connect('mysql:host=loca...', Array)
#2 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Database\Schema\BaseSchema.php(45): Cake\Database\Driver\Mysql->connect()
#3 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Database\Dialect\MysqlDialectTrait.php(63): Cake\Database\Schema\BaseSchema->__construct(Object(Cake\Database\Driver\Mysql))
#4 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(52): Cake\Database\Driver\Mysql->schemaDialect()
#5 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(42): Cake\Database\Schema\Collection->__construct(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Database\Connection.php(391): Cake\Database\Schema\CachedCollection->__construct(Object(Cake\Database\Connection), true)
#7 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\ORM\Table.php(547): Cake\Database\Connection->getSchemaCollection()
#8 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\ORM\Table.php(673): Cake\ORM\Table->getSchema()
#9 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\bake\src\Shell\Task\TemplateTask.php(288): Cake\ORM\Table->getPrimaryKey()
#10 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\bake\src\Shell\Task\TemplateTask.php(142): Bake\Shell\Task\TemplateTask->_loadController()
#11 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Console\Shell.php(532): Bake\Shell\Task\TemplateTask->main('Main')
#12 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Console\Shell.php(525): Cake\Console\Shell->runCommand(Array, false, Array)
#13 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Console\CommandRunner.php(342): Cake\Console\Shell->runCommand(Array, true)
#14 C:\xampp\htdocs\AdministradorAsistencias\vendor\cakephp\cakephp\src\Console\CommandRunner.php(164): Cake\Console\CommandRunner->runShell(Object(Bake\Shell\BakeShell), Array)
#15 C:\xampp\htdocs\AdministradorAsistencias\bin\cake.php(12): Cake\Console\CommandRunner->run(Array)
#16 {main}


2018-10-03 14:53:47 Error: [PDOException] SQLSTATE[HY000] [1045] Access denied for user 'my_app'@'localhost' (using password: YES) in C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Driver.php on line 92
Stack Trace:
#0 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Driver.php(92): PDO->__construct('mysql:host=loca...', 'my_app', 'secret', Array)
#1 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Driver\Mysql.php(106): Cake\Database\Driver->_connect('mysql:host=loca...', Array)
#2 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Schema\BaseSchema.php(45): Cake\Database\Driver\Mysql->connect()
#3 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Dialect\MysqlDialectTrait.php(63): Cake\Database\Schema\BaseSchema->__construct(Object(Cake\Database\Driver\Mysql))
#4 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(52): Cake\Database\Driver\Mysql->schemaDialect()
#5 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(42): Cake\Database\Schema\Collection->__construct(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Connection.php(391): Cake\Database\Schema\CachedCollection->__construct(Object(Cake\Database\Connection), true)
#7 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(1049): Cake\Database\Connection->getSchemaCollection()
#8 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(1003): Bake\Shell\Task\ModelTask->_getAllTables()
#9 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(213): Bake\Shell\Task\ModelTask->listAll()
#10 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(131): Bake\Shell\Task\ModelTask->getAssociations(Object(Cake\ORM\Table))
#11 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(114): Bake\Shell\Task\ModelTask->getTableContext(Object(Cake\ORM\Table), 'rounds', 'Rounds')
#12 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(101): Bake\Shell\Task\ModelTask->bake('Rounds')
#13 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Console\Shell.php(532): Bake\Shell\Task\ModelTask->main('rounds')
#14 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Console\Shell.php(525): Cake\Console\Shell->runCommand(Array, false, Array)
#15 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Console\CommandRunner.php(342): Cake\Console\Shell->runCommand(Array, true)
#16 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Console\CommandRunner.php(164): Cake\Console\CommandRunner->runShell(Object(Bake\Shell\BakeShell), Array)
#17 C:\xampp\htdocs\ProyectoInge\bin\cake.php(12): Cake\Console\CommandRunner->run(Array)
#18 {main}


2018-10-03 14:55:28 Error: [PDOException] SQLSTATE[HY000] [1045] Access denied for user 'esteban'@'localhost' (using password: YES) in C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Driver.php on line 92
Stack Trace:
#0 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Driver.php(92): PDO->__construct('mysql:host=loca...', 'esteban', 'm2LijJruLnbbFkk...', Array)
#1 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Driver\Mysql.php(106): Cake\Database\Driver->_connect('mysql:host=loca...', Array)
#2 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Schema\BaseSchema.php(45): Cake\Database\Driver\Mysql->connect()
#3 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Dialect\MysqlDialectTrait.php(63): Cake\Database\Schema\BaseSchema->__construct(Object(Cake\Database\Driver\Mysql))
#4 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(52): Cake\Database\Driver\Mysql->schemaDialect()
#5 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(42): Cake\Database\Schema\Collection->__construct(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Database\Connection.php(391): Cake\Database\Schema\CachedCollection->__construct(Object(Cake\Database\Connection), true)
#7 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(1049): Cake\Database\Connection->getSchemaCollection()
#8 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(1003): Bake\Shell\Task\ModelTask->_getAllTables()
#9 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(213): Bake\Shell\Task\ModelTask->listAll()
#10 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(131): Bake\Shell\Task\ModelTask->getAssociations(Object(Cake\ORM\Table))
#11 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(114): Bake\Shell\Task\ModelTask->getTableContext(Object(Cake\ORM\Table), 'rounds', 'Rounds')
#12 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(101): Bake\Shell\Task\ModelTask->bake('Rounds')
#13 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Console\Shell.php(532): Bake\Shell\Task\ModelTask->main('rounds')
#14 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Console\Shell.php(525): Cake\Console\Shell->runCommand(Array, false, Array)
#15 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Console\CommandRunner.php(342): Cake\Console\Shell->runCommand(Array, true)
#16 C:\xampp\htdocs\ProyectoInge\vendor\cakephp\cakephp\src\Console\CommandRunner.php(164): Cake\Console\CommandRunner->runShell(Object(Bake\Shell\BakeShell), Array)
#17 C:\xampp\htdocs\ProyectoInge\bin\cake.php(12): Cake\Console\CommandRunner->run(Array)
#18 {main}


2018-10-03 16:18:05 Warning: Warning (2): mkdir(): No such file or directory in [C:\xampp\htdocs\ProyectoInge\vendor\cakephp\bake\src\View\BakeView.php, line 108]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 159
mkdir - [internal], line ??
Bake\View\BakeView::initialize() - ROOT\vendor\cakephp\bake\src\View\BakeView.php, line 108
Cake\View\View::__construct() - CORE\src\View\View.php, line 371
Bake\Shell\Task\BakeTemplateTask::getView() - ROOT\vendor\cakephp\bake\src\Shell\Task\BakeTemplateTask.php, line 65
Bake\Shell\Task\BakeTemplateTask::generate() - ROOT\vendor\cakephp\bake\src\Shell\Task\BakeTemplateTask.php, line 88
Bake\Shell\Task\ModelTask::bakeTable() - ROOT\vendor\cakephp\bake\src\Shell\Task\ModelTask.php, line 971
Bake\Shell\Task\ModelTask::bake() - ROOT\vendor\cakephp\bake\src\Shell\Task\ModelTask.php, line 115
Bake\Shell\Task\ModelTask::main() - ROOT\vendor\cakephp\bake\src\Shell\Task\ModelTask.php, line 101
Cake\Console\Shell::runCommand() - CORE\src\Console\Shell.php, line 532
Cake\Console\Shell::runCommand() - CORE\src\Console\Shell.php, line 525
Cake\Console\CommandRunner::runShell() - CORE\src\Console\CommandRunner.php, line 342
Cake\Console\CommandRunner::run() - CORE\src\Console\CommandRunner.php, line 164
[main] - ROOT\bin\cake.php, line 12



